core/src/test/java/io/opencensus/stats/AggregationDataTest.java

```diff
@@ -46,7 +46,7 @@
   @Test
   public void testCreateDistributionData() {
     DistributionData distributionData = DistributionData.create(1, 1, 1, 1, 0, 0, 1, 0);
-    assertThat(distributionData.getSum()).isWithin(TOLERANCE).of(1);
+    assertThat(distributionData.getMean()).isWithin(TOLERANCE).of(1);
     assertThat(distributionData.getCount()).isEqualTo(1);
     assertThat(distributionData.getMean()).isWithin(TOLERANCE).of(1);
```

Comment:
It looks like the getMean assertion is redundant.

Reply:
Good catch, removed.